man/systemd.target.xml

```diff
@@ -99,6 +99,36 @@
   </refsect1>
 
   <refsect1>
+    <title>Example</title>
+
+    <example>
+      <title>Simple target</title>
+
+      <programlisting># emergency-net.target
+[Unit]
+Description=Emergency with networking
+Requires=emergency.target systemd-networkd.target
+After=emergency.target systemd-networkd.target
+AllowIsolate=yes</programlisting>
+
+      <para>When adding dependencies to other units, it's important to check if they set
+      <varname>DefaultDependencies=no</varname>. If they don't, they depend on the
+      <varname>WantedBy=</varname> value from their <varname>[Install]</varname>
+      section and therefore pull the target (and possibly the entire boot sequence)
+      as dependency.</para>
```

Comment:
This paragraph is wrong. [Install] is only used during installation. Maybe just remove the whole para.